Cargo.toml

```diff
@@ -62,6 +62,7 @@ ordered-float = { version = "3.7" }
 getrandom = { version = "0.2.10", features = ["js"], optional = true }
 
 im-rc = "15.1.0"
+rpds = "1.1.0"
```

Comment:
I switched to using `im`. Originally I used `rpds` because it seemed like `im` hasn't been maintained much (last commit 2 years ago). But don't feel strongly about it.

I had to actually install `im`, we only had `im-rc` installed, which is the non threadsafe version.